includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1944,7 +1965,7 @@ public function get_transaction_url( $order ) {
 	 * @param  string $charge_id Charge id.
 	 * @return string            Transaction details page url.
 	 */
-	private function compose_transaction_url( $charge_id ) {
+	public function compose_transaction_url( $charge_id ) {
```

Comment:
```suggestion
	protected function compose_transaction_url( $charge_id ) {
```
Since \WCPay\Payment_Method\Giropay is a child of WC_Payment_Gateway_WCPay protected access should be enough and I don't think we want to make this public unless we have to.